wiki/en/en-Compiling.md

```diff
@@ -62,13 +62,26 @@ Will build the file and make it available in `./Release/Jamulus.app`
 ## Windows
 You don't need to compile Jamulus for Windows since we provide official binaries, but if you really want to, follow the instructions below.
 
-**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio, ASIO development files
+**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio ([Build Tools 2017 or 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16) recommended), [ASIO development files](https://www.steinberg.net/en/company/developers.html)
```

Comment:
The VS link is the thank-you page, a better choice is probably this page: https://visualstudio.microsoft.com

Reply:
For me the page I linked automatically downloaded the correct installer (I only need Build Tools for example, not the whole IDE)

Reply:
Ah yes, I didn't see the download begin in the bottom bar. Perhaps we could include both that and the main downloads page (https://visualstudio.microsoft.com/downloads)

Reply:
Now both are included